test/framework/src/main/java/org/elasticsearch/index/mapper/FieldMapperTestCase2.java

```diff
@@ -228,4 +228,9 @@ private XContentBuilder mappingsToJson(ToXContent builder, boolean includeDefaul
         ToXContent.Params params = includeDefaults ? new ToXContent.MapParams(Map.of("include_defaults", "true")) : ToXContent.EMPTY_PARAMS;
         return mapping(b -> builder.toXContent(b, params));
     }
+
+    @Override
+    public void testMinimalToMaximal() {
+        // `include_defaults` includes unsupported properties in non-parametrized mappers
```

Comment:
Could you `assumeFalse` here so it logs as skipped? I don't think that makes a big difference in the long run, but it makes me feel a little better about doing nothing in a test.